### ISSUE TITLE
The expand/collapse of a configuration session is undone when changing tabs.

### ISSUE THREAD

**Issue by lauroguedes** [gantry/gantry5#2046]
I would suggest that when I retracted a configuration session it would still retracted even if I change tabs, this greatly improves usability.

**Comment by w00fz** [comment]
What do you mean when changing tabs is retracted?
Are you referring to 2 different sites running gantry on 2 different tabs?

**Comment by lauroguedes** [comment]
Maybe for gif you undertand the problem.

![exemplo-graphic](https://user-images.githubusercontent.com/3677780/28596205-c427a7d6-716d-11e7-8696-538a7f4b1f2a.gif)

**Comment by lauroguedes** [comment]
when I change tab it don't fixe the collapse

**Comment by w00fz** [comment]
Try clearing your cookies, it might be sort of 'stuck'.
Also which platform is this on?

**Comment by w00fz** [comment]
Oh so this is this working on reload but not when switching tab?

**Comment by lauroguedes** [comment]
Then the problem happens both when changing the tab and when reloading the page. I'm doing the Joomla test. With you there it is working normal?

**Comment by w00fz** [comment]
Yeah it preserves the status of the panels for me on both reload and tabs switching.
I would try clearing out the cookies. Specifically the `g5-collapsed` cookie

**Comment by lauroguedes** [comment]
I'm going to do a more elaborate tests and in other browsers to see, because it is still giving problem even cleaning the cookies

**Comment by mahagr** [comment]
@lauroguedes any updates on this issue?

**Comment by kat05** [comment]
I am testing on Joomla 3.8 beta 4 with Gantry and Hydrogen 5.4.17.

After both reload and changing tabs, any collapsed items are expanded again.
